server/game/cards/11.4-MoD/GreyGhost.js

```diff
@@ -0,0 +1,63 @@
+const DrawCard = require('../../drawcard');
+
+class GreyGhost extends DrawCard {
+    constructor(owner, cardData) {
+        super(owner, cardData);
+
+        this.tracker = new ShadowsTracker(this.game, this);
+    }
+
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Choose character(s)',
+            cost: ability.costs.kneelSelf(),
+            handler: () => {
+                this.game.promptForSelect(this.controller, {
+                    numCards: this.tracker.hasComeOutOfShadowsThisPhase ? 2 : 1,
+                    activePromptTitle: this.tracker.hasComeOutOfShadowsThisPhase ? 'Select 2 characters' : 'Select a character',
+                    source: this,
+                    cardCondition: card => card.location === 'play area' && card.getType() === 'character',
+                    onSelect: (player, cards) => this.onSelect(player, cards),
+                    onCancel: (player) => this.cancelSelection(player)
+                });
+            }
+        });
+    }
+
+    onSelect(player, cards) {
+        if(!Array.isArray(cards)) {
```

Comment:
If you add `multiselect: true` to the target definition, `cards` (or `context.target` if done via the target API) will be an array irrespective of the number of targets chosen.